package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "gitops-secrets",
+  "version": "0.0.1",
   "author": "Ryan Blunden <ryan.blunden@doppler.com>",
   "description": "GitOps encrypted secrets workflow",
   "repository": {
     "type": "git",
-    "url": "git://github.com/doppler-university/gitops-secrets.git"
+    "url": "https://github.com/doppler-university/gitops-secrets.git"
```

Comment:
It's actually https://github.com/DopplerUniversity/gitops-secrets-nodejs, right?